app/assets/javascripts/components/login_form.es6.jsx

```diff
@@ -20,12 +21,14 @@ class LoginForm extends React.Component {
     return (
       <div style={this.styles.container}>
         <input
+          autoFocus={true}
           placeholder={'email'}
           style={this.styles.input}>
         </input>
         <input
           placeholder={'password'}
-          style={this.styles.input}>
+          style={this.styles.input}
+          type={'password'}>
```

Comment:
Is wrapping this in {'s just a convention? Just 'type="password"' would work as well right?